website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+
+This guide describes how Karpenter interacts with external resources, and modifies cluster behavior. 
+
+## Resources Dependencies
+
+### Cluster on AWS
+
+Currently, Karpenter creates nodes (EC2 Instances) through the AWS cloud provider module, the only such module implemented. Clusters on other cloud providers are contemplated, but are not currently supported. 
+
+Theoretically, a control plane could be hosted separately from Karpenter provisioned AWS instances. However, this complicates identity and authorization substantially. The Karpenter controller (i.e., agent) would need authorization with AWS APIs to provision new instances, and permissions with the control plane to add the new nodes. 
+
+Additionally, the intricacies of securing control plane communication between clouds presents another significant problem. 
+
+The getting started guide covers [configuring EKS to accept newly provisioned instances as nodes](../getting-started/#create-the-karpenternode-iam-role) (see, *KarpenterNodeRole* and *system:bootstrappers*). If using a self-managed (or alternatively provided) control plane, this must be manually handled. Consult the [*User Data - Autoconfigure* section](../cloud-providers/aws/launch-templates/#user-data---autoconfigure) of *Launch Templates* for more information on joining nodes to a cluster.
+
+### Minimum Kubernetes Version
+
+Karpenter supports the [same set of Kubernetes versions as EKS supports](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) for new clusters. Karpenter strongly encourages a posture of promptly adopting new versions of Kubernetes.
+
+### Service Account
+    - `Karpenter` Service Account
+
+The `Karpenter` service account needs sufficient cluster permissions to, for example, add new nodes and bind pods to nodes. Review the ["karpenter-controller" `role` and `ClusterRole`](rbac.yaml) from the default helm chart to understand what permissions the Karpenter service account needs. The Karpenter helm chart includes other roles (`karpenter-webhook`), but the "karpenter-controller" resources require uniquely broad access.
+
+In part:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: karpenter-controller
+rules:
+[...]
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - create
+[...]
+```
+
+
+### IAM Roles
+- `Karpenter` Role
+- `KarpenterNode` Role
+
+Generally, Karpenter needs to authenticate with the AWS EC2 API to create new instances. This is best done with IRSA, IAM Roles for Service Accounts. IRSA naturally extends the `Karpenter` service account to handle interacting with AWS APIs. The getting started guide describes creating a cluster service account mapped to an IAM role with sufficient permissions. 
+
+Karpenter may alternatively be configured to authenticate with AWS using the `kube2iam` package, or with locally stored Access Keys. Both practices are discouraged. 

```

Comment:
Why are they discouraged?

Reply:
added a sentence describing this. I'm under the impression that the usual response to "kube2iam" is "people still use that?" -- although I'm not sure if the attitude is the same outside amazon.